README.md

```diff
@@ -96,6 +96,15 @@ Supporting contacts this way has a number of downsides:
   * Feature detection is harder, and would likely need a new API such as `HTMLInputElement.isTypeSupported()` mimicking the [`MediaRecorder`](https://www.w3.org/TR/mediastream-recording/#dom-mediarecorder-istypesupported). Unlike existing specializations of file pickers, contacts would be unlikely to gracefully degrade to a general picker.
   * Extensibility is harder, as it would rely on additional parameters being passed to the mime type.
 
+#### Previous Standardization Attempts
+There have been multiple previous attempts to bring a Contacts API to the web:
+  * https://wiki.mozilla.org/WebAPI/ContactsAPI
```

Comment:
This doesn't seem to have been on the standards track. Probably still worth mentioning given that a lot of thought went into this, but it's not a previous standardization attempt. Maybe a separate paragraph for proprietary attempts?